src/ResourceManager/ContainerRegistry/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Current Release
 * Fix issue with Default Resource Group in CloudShell
```

Comment:
@djyou it looks like you need to pull in the latest changes from the `preview` branch since this line should be under the `Version 1.0.3` header